infra/modules/curi/s3_downloads/iam.tf

```diff
@@ -0,0 +1,91 @@
+resource "aws_iam_role" "downloads_role" {
+  name = "s3_downloads_role"
+
+  assume_role_policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Action = "sts:AssumeRole"
+        Effect = "Allow"
+        Sid    = ""
+        Principal = {
+          AWS = "arn:aws:iam::${var.account_id}:root"
```

Comment:
does this allow any IAM user in the admin account to assume this role? or is this a policy being created in the admin account that we can then manually assign to certain IAM users? ...or something else and I'm completely misunderstanding it?

Reply:
or are you saying you went into the admin account and then manually created a group there and added this policy ARN from the test/modl/prod accounts?

Reply:
It only allows users who have the sts:AssumeRole permissions, I created a group in the admin account with the policy
```
{
  "Version": "2012-10-17",
  "Statement": [
    {
      "Sid": "VisualEditor0",
      "Effect": "Allow",
      "Action": "sts:AssumeRole",
      "Resource": [
        "arn:aws:iam::<test id>:role/s3_downloads_role",
        "arn:aws:iam::<modl id>:role/s3_downloads_role",
        "arn:aws:iam::<prod id>:role/s3_downloads_role"
      ]
    }
  ]
}
```
so any user in that group can assume the role to access only the download bucket in each environment + cloudfront invalidation

Reply:
I tested it by adding the s3_test user I created to the group and was able to create files in the downloads-test bucket, but my actual user account in admin that doesn't have sts:AssumeRole permissions and isn't in that group couldn't create files in the downloads-test bucket

Reply:
great, thanks for the clarification.  it looks like this needs to be `rebased` before it can be merged into development though

Reply:
![image](https://user-images.githubusercontent.com/5436635/111845410-267eed80-88db-11eb-8a35-73b723179406.png)